Integrations/Panorama/Panorama.py

```diff
@@ -1792,45 +1792,50 @@ def panorama_get_url_category(url):
     return s.split(' ')[1]
 
 
-def populate_url_filter_category_from_context(url_category_hr):
-    url_filter_category = demisto.dt(demisto.context(),
-                                     'Panorama.URLFilter(val.Category === "{0}")'.format(url_category_hr['Category']))
-
+def populate_url_filter_category_from_context(category):
```

Comment:
did you test it with empty and with a filter?
and why are we populating from context and not from argument?

Reply:
yes. because context may have a category with 1 or more urls, and i want to union all the urls from the context and from the command arguments to be under the same category